docs/command_line_tool/alternate_install.md

```diff
@@ -23,3 +25,20 @@ Phylum is written in Rust, so you'll need a recent Rust installation to build it
    ```sh
    cargo install --path cli
    ```
+
+## Install with phylum-ci
+
+See the [phylum-ci](https://github.com/phylum-dev/phylum-ci) project for details.
+
+```sh
+pipx install phylum-ci
+phylum-init
```

Comment:
```suggestion
pip install phylum
phylum-init
```
The repository is named `phylum-ci`. The package is named `phylum`. The script entry point is named `phylum-init`. Not confusing at all 😛.
Even though I greatly prefer `pipx` for installing packages that provide script entry points, that assumes one more pre-requisite for the user. Sticking with plain old `pip` is best for this form of the documentation. More detail is provided in the links to the package/repo, provided above.

Reply:
Is Python stuff automatically added to the $PATH? Or does that require manual steps.

Reply:
The short answer is yes.

The script entry points will be automatically available on the `$PATH`, whether the install was with `pip` or `pipx`. With a `pip` based install, if the package was installed in a Python virtual environment, then that environment needs to be active for the scripts to be on the `$PATH`.

Reply:
> Even though I greatly prefer `pipx` for installing packages that provide script entry points, that assumes one more pre-requisite for the user. Sticking with plain old `pip` is best for this form of the documentation. More detail is provided in the links to the package/repo, provided above, for intrepid users.

This is the alternate install page, so we are already dealing with "intrepid users". And `pipx` has significant advantages here. I still believe `pipx` deserves the recommendation on this page

Reply:
> Is Python stuff automatically added to the $PATH? Or does that require manual steps.

This is one of the benefits of `pipx` since it will handle this. But even with `pip`, there should be a warning printed if the script directory isn't in your path when the package is installed.